### DISCUSSION TITLE
1.5.0

### DISCUSSION THREAD

**Discussion by villebro** [apache/superset#19882] [General]
Hello Community,

The Apache Superset team is pleased to announce that Superset
1.5.0 has just been released.

Apache Superset is a modern, enterprise-ready business intelligence web application

The official source release:

https://www.apache.org/dist/superset/1.5.0

The Pypi package:

https://pypi.org/project/apache-superset/1.5.0

If you have any usage questions, or have problems when upgrading or
find any problems about enhancements included in this release, please
don't hesitate to let us know by sending feedback to this mailing
list.

---------------------------------------------------------------------
To unsubscribe, e-mail: general-unsubscribe@apache.org
For additional commands, e-mail: general-help@apache.org
--------------------------

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.5.0'>1.5.0</a>.</em>

**Comment by nikita-karatun** [comment]
I'm having a warning with the following message.

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Was the release branch created in the fork and never pushed back into the original repo?
